eap_backend/eap_api/views.py

```diff
@@ -641,12 +641,45 @@ def comment_list(request, assurance_case_id):
         return Response(serializer.data)
 
     elif request.method == "POST":
-        serializer = CommentSerializer(data=request.data)
+        data = request.data.copy()
+        data["assurance_case_id"] = (
+            assurance_case_id  # Ensure assurance_case_id is set in the data
+        )
+        serializer = CommentSerializer(data=data)
         if serializer.is_valid():
             # Ensure the author is set to the current user
             serializer.save(author=request.user)
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    return JsonResponse(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def comment_detail(request, pk):
```

Comment:
Given that we're out of "crunch-mode", please at least at some unit tests for the new backend code. At least at the `test_view` level

Reply:
@cptanalatriste this is where I was hoping you'd be able to help

Reply:
@RichGriff of course! I can add a couple of tests to the branch